servers/src/main/java/tachyon/worker/block/BlockLockManager.java

```diff
@@ -60,24 +58,22 @@
   /** A hashing function to map blockId to one of the locks */
   private final HashFunction mHashFunc = Hashing.murmur3_32();
 
-  public BlockLockManager(BlockMetadataManager metaManager) {
-    mMetaManager = Preconditions.checkNotNull(metaManager);
+  public BlockLockManager() {
     for (int i = 0; i < NUM_LOCKS; i ++) {
       mLockArray[i] = new ClientRWLock();
     }
   }
 
   /**
-   * Locks a block if it exists, throws NotFoundException otherwise.
+   * Locks a block. Note that, lock striping is used so even this block does not exist, a lock id
+   * is still returned.
```

Comment:
Right, this change is to simplify the logic: otherwise we need to meta-lock `mMetaManager` in this method to check the existence of the given block.